index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  <title>My Portfolio</title>
+</head>
+<body>
+
+  <header>
+
+    <div id="settings">
+      <div id="settings-date">
+
+      </div>
+      
+      <div class="settings-fa">
+        <i class="fas fa-signal"></i>
+        <i class="fas fa-wifi"></i>
+        <i class="fas fa-battery-full"></i>
+      </div>
+    </div>
+
+    <div id="union">
+      <img src="images/Icons/Icon - Menu.png" alt="union">
+    </div>
+
+    
+  </header>
+
+  <section class="headline">
+
+    <div>
+      <img id="headline-bg" src="images/Header-llustration-desktop@2x.png" alt="background-image">
+    </div>
+
+    <div class="headline-text">
+      <h1>
+        Hey there. 
+        <br>
+        I’m Didy.
+        <br>
+        <span class="green"> I’m a Full Stack developer from Microverse</span>
+      </h1>
+
+      <h2>
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.
+      </h2>
+
+      <div class="social">
+        <a href="https://twitter.com/diarisdiakite">
+          <i class="fab fa-twitter"></i>
+        </a>
+        <a href="https://www.linkedin.com/diarisdiakite/">
+          <i class="fab fa-linkedin-in"></i>
+        </a>
+        <a href="https://www.medium.com/#">
+          <i class="fab fa-medium"></i>
+        </a>
+        <a href="https://www.github.com/diarisdiakite">
+          <i class="fab fa-github"></i>
+        </a>
+        <a href="https://www.angellist.com/#">
+          <i class="fab fa-angellist"></i>
+        </a>
+      </div>
```

Comment:
- [ ] This is a list of social media links, there what should be used here is a `<ul>` stage instead of `<div>` and the `<a>` tags should be wrapped inside `<li>` tags, it's more semantically proper and will improve accessibility.

Reply:
Done @ouasamine